platform/web/request.go

```diff
@@ -73,19 +73,20 @@ func DecodeWithJSONSchema(request *http.Request, model interface{}, filePath str
 		requestContentErrorDetails := make(ErrorDetails)
 
 		for _, desc := range result.Errors() {
-			if desc.Type() == "required" {
-				property, ok := desc.Details()["property"]
-				if !ok {
-					property = desc.Field()
+			propertyName := desc.Field()
+			if propertyName == "(root)" {
+				details, ok := desc.Details()["property"]
+				if ok {
+					propertyName = fmt.Sprintf("%v", details)
 				}
+			}
```

Comment:
After reading the tests, this seems to be working but I'm not sure to understand how it does so. Would there be a way to clarify the use of the check ? 
If I didn't know beforehand that it was to make it work for a `map[string]string`  property like `localized_string` I would not be able to understand this part of the code easily.

I know we're likely to change this as soon as Kevin shows his new solution, so it's not too much of an issue since this should not be long-lived if I understood well 😄

Reply:
Ah this was already merged, I should've refreshed.

Reply:
I'm not sure what you don't get about this part of the code? I use a method to get the property name of the invalid attribute, but if this equals to "(root)" I call another method to see if that has a value and update it accordingly?